### DISCUSSION TITLE
Is there a way to change the container owner:group ids?

### DISCUSSION THREAD

**Discussion by CaptainPackers** [kromitgmbh/titra#150] [Q&A]
Apparently the container owner:group id numbers are 999:999, which map to weird owner and group on my persistent database volume. Is there a way these can be set to other values?

Thanks

**Comment by faburem** [comment]
I will do some research and  see what we can do in the docker image definition!

**Comment by faburem** [comment]
Apparently you can override the default user settings in a container by specifying --user UID:GUID in the docker run command (this works also in docker-compose). I have found this answer on stackoverflow with more details:
https://stackoverflow.com/questions/56844746/how-to-set-uid-and-gid-in-docker-compose

**Reply by CaptainPackers** [reply]
Thanks for the help
